modules/toast/react/lib/Toast.tsx

```diff
@@ -75,19 +75,14 @@ const Message = styled('div')({
 });
 
 export default class Toast extends React.Component<ToastProps> {
-  static defaultProps = {
-    icon: checkIcon as CanvasSystemIcon, // needed for TS2742 - https://github.com/microsoft/TypeScript/issues/29808
-    iconColor: colors.greenApple400,
-  };
-
   public render() {
     const {
+      icon = checkIcon as CanvasSystemIcon, // needed for TS2742 - https://github.com/microsoft/TypeScript/issues/29808
+      iconColor = colors.greenApple400,
+      transformOrigin = {horizontal: 'center', vertical: 'top'} as const,
```

Comment:
What was this added for?

Reply:
at top in the prop description it has a default, which is why i added it as a default when I destructured it

Reply:
It's because that's the default of `Popup`. I think we can document what the sub-components defaults are, but I don't think we should re-establish them here.

Reply:
ah makes sense, thanks for clarifying!